Add MV3 support blurb to selected pages. (#6085)
GoogleChrome/developer.chrome.com: site/en/_partials/extensions/mv3-support.md

{% Aside %}

> Manifest V3 is supported generally in Chrome 88 or later. For extension features added in later Chrome versions, see the descriptions of specific APIs or their members for support information. If your extension requires a specific API, you can specify [a minimum chrome version](/docs/extensions/mv3/manifest/minimum_chrome_version/) in the manifest file.

Reviewer: Maybe link to the reference docs

```suggestion
Manifest V3 is supported generally in Chrome 88 or later. For extension features added in later Chrome versions, see the [API reference documentation](/docs/extensions/reference/) for support information. If your extension requires a specific API, you can specify [a minimum chrome version](/docs/extensions/mv3/manifest/minimum_chrome_version/) in the manifest file.
```
